client/src/components/Form/Form.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components/macro'
+import { useState } from 'react'
+
+const Form = ({ onAddQuestion }) => {
+  const [question, setQuestion] = useState('')
+
+  const changeQuestionHandler = event => {
+    setQuestion(event.target.value)
+  }
+
+  const submitForm = () => {
+    onAddQuestion(question)
+  }
+
+  return (
+    <StyledForm onSubmit={submitForm}>
+      <StyledLabel htmlFor="newQuestion">Question</StyledLabel>
+      <StyledInput
+        type="text"
+        id="newQuestion"
+        name="newQuestion"
+        placeholder="Please enter your question"
+        onChange={event => changeQuestionHandler(event)}
+      />
+      <StyledButton type="submit">Save</StyledButton>
```

Comment:
No need to set the type to submit as any button within a form element functions as a submit button.